docs/sdks/server-sdks/node/quickstart.mdx

```diff
@@ -0,0 +1,313 @@
+
+### Initialize the SDK
+
+[Create an SDK key](/sdks/sdk-keys) if you don't already have one. 
+
+First, initialize the SDK using your SDK key. The SDK key is different from the project API key. 
+
+
+```js
+import init from "@eppo/js-server-sdk";
+
+await init({apiKey: 'SDK_KEY'});
+``` 
+
+:::note
+The SDK key is different from the project API key. You can find your SDK key in the [SDK Keys section of the Eppo interface](https://eppo.cloud/configuration/environments/keys).
+:::
+
+### Assign a variant
+
+Once initialized, the SDK uses a singleton pattern, making it accessible from anywhere in your application. To make assignments, import the SDK and use `getInstance()` to access the SDK instance.
+
+### How assignments work
+
+The SDK periodically retrieves configuration rules from the Eppo server that define how subjects should be allocated to variants. When you call an assignment function, the SDK evaluates these rules locally without making additional network requests.
+
+Each assignment requires:
+
+- **Flag Key**: Identifies which set of configuration rules to use
+- **Subject Key**: A unique identifier for the subject (usually a user ID)
+- **Subject Attributes**: Optional key-value pairs containing additional information used for rule evaluation
+- **Default Value**: Fallback value if assignment fails or rules don't match
+
+```js
+import * as EppoSdk from "@eppo/js-server-sdk";
+
+// Get SDK instance
+const eppoClient = EppoSdk.getInstance();
+
+// Define assignment parameters
+const featureKey = "my-neat-feature";
+const subjectKey = getCurrentUser() || "anonymous";
+const subjectAttributes = {
+    'country': user.country,
+    'device': user.device,
+};
+const defaultValue = "default-value";
+
+// Make an assignment
+const variation = eppoClient.getStringAssignment(
+  featureKey,
+  subjectKey,
+  subjectAttributes,
+  defaultValue
+);
+```
+
+### Assignment Types
+
+The SDK provides different assignment functions based on the type of value you need:
+
+| Function | Return Type |
+|----------|-------------|
+| `getStringAssignment()` | String |
+| `getBooleanAssignment()` | Boolean |
+| `getJSONAssignment()` | JSON object |
+| `getIntegerAssignment()` | Integer |
+| `getNumericAssignment()` | Float |
+
+:::note
+See more details about assignment functions in the [Assignments](/sdks/client-sdks/javascript/assignments) page.
+:::
+
+### Using Assignments
+
+After receiving an assignment, your application should implement logic to modify the user experience accordingly:
+
+```javascript
+// Example using a boolean assignment
+const variation = eppoClient.getStringAssignment(
+  featureKey,
+  subjectKey,
+  subjectAttributes,
+  defaultValue
+);
+
+// Render different components based on assignment
+if (variation === "landing-page-a") {
+    return <LandingPageA />
+} else if (variation === "landing-page-b") {
+    return <LandingPageB />
+} else {
+    return <LandingPageC />
+}
```

Comment:
JSX doesn't make sense in the context of a Node.js application, as this will be on the back end. Use a back-end example instead.
```ts
let algorithm;
if (variation === 'new-algorithm') {
  algorithm = ALGORITHM.BIASED_RANDOM_WALK;
} else {
  algorithm = ALGORITHM.RANDOM_WALK;
}
const result = runAlgorithm(algorithm);
```